src/templates/header/index.jsx

```diff
@@ -89,7 +89,7 @@ Header.defaultProps = {
 };
 
 Header.propTypes = {
-  siteName: PropTypes.string.isRequired,
+  siteName: PropTypes.node.isRequired,
```

Comment:
This fixes an error in console when an image or something is passed to the site name. The code has been working but the prop just sends an error during dev mode.